classes/SCLOrkWatch.sc

```diff
@@ -0,0 +1,203 @@
+/*
+Usage:
+SCLOrkWatch.new;
+
+Optionally, user may define a list of target times to get a blink warning.
+Blinking will start 5 seconds before target:
+
+SCLOrkWatch.new(["0:13", "0:30", "01:11"]);
+
+Optionally, user can enter list in the format "target times & actions":
+
+[
+"0:05", { "wow".postln },
+"0:10", { "boo".posltn },
+]
+
+The function will be evaluated at the corresponding time.
+
+*/
+
+SCLOrkWatch {
+
+	var win;
+	var secondsView, minutesView, separatorView;
+	var winWidth = 400, winHeight = 160;
+	var timeInSeconds = 0;
+	var mm = 0, ss = 0, updateMinSec, to_seconds;
+	var formatString;
+	var font, t;
+	var blinkFunction, blinkTimer, blinkDuration = 5;
+	var <>blinkList;
+	var blinkEndAction;
+
+	*new { arg list; ^super.new.init(list); }
+
+	init { arg list;
+
+		// blinkList stores *blink start times*, not target times
+		// blink start time is (target - blinkDuration)
+
+		// new blink list function
+		blinkList = {
```

Comment:
Confusing comment, makes it seem like blinkList is a function. Also consider using something like Array.fill(size, function) to make it more clear you are making an array.